docs/source/configuration/entity-caching.mdx

```diff
@@ -211,7 +211,7 @@ telemetry:
       cache: # Cache instruments configuration
         apollo.router.operations.entity.cache: # A counter which counts the number of cache hit and miss for subgraph requests
           attributes:
-            entity.type: true # Include the entity type name. default: false
+            graphql.entity.type: true # Include the entity type name. default: false
```

Comment:
I think this needs to be `graphql.type.name` to align with the attributes we already have.